### PR TITLE
~markdown-cards

### PR DESCRIPTION
**Pull request by taravancil**
This converts the page descriptions from raw markdown into "plain text", so that social cards don't display raw markdown

Before:
Screen Shot 2019-03-22 at 9 26 13 AM

After:
Screen Shot 2019-03-22 at 9 26 01 AM

TODOs:
- [x] decide whether this should happen within individual routes or inside of `render`
- [x] make sure there aren't other markdown fields

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -95,6 +95,6 @@ module.exports = {
   getProject: (domain) => getFromCacheOrApi(domain, projectCache, getProjectFromApi),
   getTeam: (url) => getFromCacheOrApi(url, teamCache, getTeamFromApi),
   getUser: (login) => getFromCacheOrApi(login, userCache, getUserFromApi),
-  getCollection: url => getFromCacheOrApi(url, collectionCache, getCollectionFromApi),
+  getCollection: (url) => getFromCacheOrApi(url, collectionCache, getCollectionFromApi),
```
Comment (taravancil) [inline server/api.js]: linter fix

File: shrinkwrap.yaml [hunk 2]
```diff
@@ -7525,13 +7549,21 @@ packages:
     dev: false
     resolution:
       integrity: sha1-dFz0GlNZexj2iImFREBe+isG2Q8=
+  /lodash.assignin/4.2.0:
```
Comment (sarahzinger) [inline shrinkwrap.yaml]: are these changes intentional?

Comment (taravancil) [reply shrinkwrap.yaml]: are changes to shrinkwrap ever intentional? lolll no they're not 🙃

**Comment by taravancil** [comment]
@sarahzinger that's an interesting idea and a good point! I think to make that work we'd need to store all markdown fields twice: once as raw markdown, once as text content. That seems reasonable to me, and probably ultimately the Right Thing, but my gut feeling says that maybe this is an ok solution for now and maybe forever?
